llvm/lib/Target/AMDGPU/SOPInstructions.td

```diff
@@ -326,8 +326,10 @@ def S_XNOR_SAVEEXEC_B64 : SOP1_64 <"s_xnor_saveexec_b64">;
 
 } // End hasSideEffects = 1, Uses = [EXEC], Defs = [EXEC, SCC]
 
-def S_QUADMASK_B32 : SOP1_32 <"s_quadmask_b32">;
-def S_QUADMASK_B64 : SOP1_64 <"s_quadmask_b64">;
+def S_QUADMASK_B32 : SOP1_32 <"s_quadmask_b32",
+[(set i32:$sdst, (int_amdgcn_s_quadmask i32:$src0))]>;
```

Comment:
Nit: we would normally indent this line two spaces (although there are no official tablegen formatting rules). Same below.

Reply:
I don't think it matters too much. I hope that one day we will have a tablegen-format tool to do it automatically.